packages/rest/src/__tests__/unit/coercion/paramStringToInteger.unit.ts

```diff
@@ -3,30 +3,30 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {ParameterLocation} from '@loopback/openapi-v3';
+import {ParameterLocation, ParameterObject} from '@loopback/openapi-v3';
 import {RestHttpErrors} from '../../../';
 import {test} from './utils';
 
-const INT32_PARAM = {
+const INT32_PARAM: ParameterObject = {
   in: <ParameterLocation>'path',
```

Comment:
Now that the entire PARAM constant is cast to `ParameterObject`, I think it is no longer necessary to cast the `in` value too. Could you please check?

```suggestion
  in: 'path',
```

The same comment applies to other similar places below.